### DISCUSSION TITLE
Deploy on Hostinger

### DISCUSSION THREAD

**Discussion by doublegarts** [serverpod/serverpod#601] [Q&A] [answered]
Hello, I have a hostinger.com plan.
What are the steps to deploy my Serverpod app to it?
My hosting plan also supports only MySql database.

**Comment by vlidholt** [comment, marked as answer]
It looks like Hostinger only supports web hosting and is focused on Wordpress. It may not be a good choice for hosting a Serverpod.

**Reply by fischerscode** [reply]
They seem to also support VPS.
